### PR TITLE
chore: use colors from @dhis2/ui-core

### PR DESCRIPTION
**Pull request by edoardo**
Removed the local colors module.

### CODE REVIEW

**Comment by edoardo** [comment]
There are a couple more hardcoded colors in App.css.

**Comment by jenniferarnesen** [comment]
The text color for the spacer item looks a little too faded. Other than that, looks good.

File: src/components/Item/SpacerItem/Item.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-import { colors } from '../../../modules/colors';
+import { colors } from '@dhis2/ui-core';
 import ItemHeader from '../ItemHeader';
 
 const style = {
     margin: '21px 28px',
     fontSize: '14px',
     lineHeight: '18px',
-    color: colors.charcoalGrey,
+    color: colors.grey400,
```
Comment (cooper-joe) [inline src/components/Item/SpacerItem/Item.js]: Use grey600 instead of grey400 here

**Review by cooper-joe — CHANGES_REQUESTED** [review]
Just a single change, other than that looks good.

**Review by jenniferarnesen — APPROVED** [review]

**Review by jenniferarnesen — APPROVED** [review]

**Review by cooper-joe — APPROVED** [review]

**Pull request merged**